ktor-server/ktor-server-test-suites/jvm/src/io/ktor/server/testing/suites/ContentTestSuite.kt

```diff
@@ -235,6 +236,20 @@ public abstract class ContentTestSuite<TEngine : ApplicationEngine, TConfigurati
         }
     }
 
+    @Test
+    fun testReceiveBlocking() {
```

Comment:
looks like it is not related to formatting, but ok for me

Reply:
That wasn't intentional! Thanks!